src/libs/konnectors/stopKonnectorJob.ts

```diff
@@ -0,0 +1,61 @@
+import Minilog from '@cozy/minilog'
+
+import CozyClient, { Q } from 'cozy-client'
+
+import { TIMEOUT_KONNECTOR_ERROR } from '/libs/Launcher'
+
+const log = Minilog('stopKonnectorJob')
+
+export const stopKonnectorJob = async (
+  client: CozyClient,
+  jobId: string
+): Promise<void> => {
+  try {
+    await stopJob(client, jobId)
+    await findJobStopTriggerAndRemove(client, jobId)
+  } catch (e) {
+    log.error('Error while trying to stop job ${jobId}')
+    log.error(e)
+  }
+}
+
+/**
+ * Mark the given job as "context deadline exceeded"
+ */
+const stopJob = (client: CozyClient, jobId: string): Promise<unknown> => {
+  return client.save({
+    _type: 'io.cozy.jobs',
+    _id: jobId,
+    _rev: true, // to force an update on Job collection
```

Comment:
same as here https://github.com/cozy/cozy-home/pull/1897#discussion_r1130794135 

we should not do that and pass the real rev 🙏

Reply:
I was trying to avoid to get the job for performance reason. But let's do the get